app/src/main/java/com/spencerpages/collections/LargeCents.java

```diff
@@ -0,0 +1,120 @@
+            {"Liberty Cap", R.drawable.a1794_cent_obv_venus_marina},
+            {"Draped Bust", R.drawable.a1797cent_obv},
+            {"Capped Bust", R.drawable.annc_us_1813_1c_classic_head_cent},
+            {"Matron Coronet", R.drawable.a1819centrev},
+            {"Young Coronet", R.drawable.a1837_cent_obv},
+            {"Petite Braided Hair", R.drawable.a1839},
+            {"Mature Braided Hair", R.drawable.a1855},
+    };
+
+    private static final HashMap<String, Integer> COIN_MAP = new HashMap<>();
+
+    static {
+        for (Object[] coinData : COIN_IDENTIFIERS) {COIN_MAP.put((String) coinData[0], (Integer) coinData[1]);}
+    }
+
+    private static final int REVERSE_IMAGE = R.drawable.a1819_cent_obv;
+
+    @Override
+    public String getCoinType() {
+        return COLLECTION_TYPE;
+    }
+
+    @Override
+    public int getCoinImageIdentifier() {
+        return REVERSE_IMAGE;
+    }
+
+    @Override
+    public int getStartYear() {return START_YEAR;}
+
+    private static final Integer START_YEAR = 1793;
+    private static final Integer STOP_YEAR = 1855;
+
+    @Override
+    public int getStopYear() {return STOP_YEAR;}
+
+    @Override
+    public int getCoinSlotImage(CoinSlot coinSlot) {
+        Integer slotImage = COIN_MAP.get(coinSlot.getIdentifier());
+        return (slotImage != null) ? slotImage : (int) COIN_IDENTIFIERS[0][1];
+    }
+
+    @Override
+    public void getCreationParameters(HashMap<String, Object> parameters) {
+        parameters.put(CoinPageCreator.OPT_EDIT_DATE_RANGE, Boolean.FALSE);
+        parameters.put(CoinPageCreator.OPT_START_YEAR, START_YEAR);
+        parameters.put(CoinPageCreator.OPT_STOP_YEAR, STOP_YEAR);
+
+        parameters.put(CoinPageCreator.OPT_CHECKBOX_1, Boolean.TRUE);
+        parameters.put(CoinPageCreator.OPT_CHECKBOX_1_STRING_ID, R.string.include_bust);
+
+        parameters.put(CoinPageCreator.OPT_CHECKBOX_2, Boolean.TRUE);
+        parameters.put(CoinPageCreator.OPT_CHECKBOX_2_STRING_ID, R.string.include_coronet);
+    }
+
+    @Override
+    public void populateCollectionLists(HashMap<String, Object> parameters, ArrayList<CoinSlot> coinList) {
+        Integer startYear = (Integer) parameters.get(CoinPageCreator.OPT_START_YEAR);
+        Integer stopYear = (Integer) parameters.get(CoinPageCreator.OPT_STOP_YEAR);
+        Boolean showbust = (Boolean) parameters.get(CoinPageCreator.OPT_CHECKBOX_1);
+        Boolean showcoronet = (Boolean) parameters.get(CoinPageCreator.OPT_CHECKBOX_2);
+
+        int coinIndex = 0;
+
+        for (Integer i = startYear; i <= stopYear; i++) {
+            String date = String.format("%2d", i );
+            String mdate = String.format("%n%2d", i );
+            if (showbust) {
+                if (i == 1793) {
+                    coinList.add(new CoinSlot("Flowing Hair Chain Reverse", date, coinIndex++));
+                    coinList.add(new CoinSlot("Flowing Hair Wreath Reverse", date, coinIndex++));}
```

Comment:
If you change the comment above, change this also
```suggestion
                    coinList.add(new CoinSlot("Flowing Hair Chain", date, coinIndex++));
                    coinList.add(new CoinSlot("Flowing Hair Wreath", date, coinIndex++));}
```